kernel/src/device/pty/master.rs

```diff
@@ -120,7 +120,7 @@ impl FileIo for PtyMaster {
                 // TODO: Deal with `open()` flags.
                 let slave = {
                     let slave_name = {
-                        let devpts_path = super::DEV_PTS.get().unwrap().abs_path();
+                        let devpts_path = "/dev/pts";
```

Comment:
Hardcoding the path is not right. Theoretically, devtmpfs may be mounted at any locations.